.vscode/launch.json

```diff
@@ -1,29 +0,0 @@
-{
```

Comment:
These files shouldn't be removed.  They are needed to build and debug the extension from within VS Code.